src/actions/websocket.js

```diff
@@ -457,6 +460,28 @@ function handleChannelDeletedEvent(msg, dispatch, getState) {
     }
 }
 
+function handleChannelUpdatedEvent(msg, dispatch, getState) {
+    try {
+        const channel = msg.data ? JSON.parse(msg.data.channel) : null;
+        const entities = getState().entities;
+        const {currentChannelId} = entities.channels;
+        if (channel) {
+            dispatch({
+                type: ChannelTypes.RECEIVED_CHANNEL,
+                data: channel
+            });
+
+            if (currentChannelId === channel.id) {
+                // Emit an event with the channel received as we need to handle
+                // the changes without listening to the store
+                EventEmitter.emit(WebsocketEvents.CHANNEL_UPDATED, channel);
+            }
+        }
+    } catch (err) {
+        // do nothing
+    }
```

Comment:
Is the try catch just for the JSON parsing? If so, maybe just put it around that and return in the catch

Reply:
will do